Advantzware/Advantzware
# 62005 AutoLogout - Should run on close of any Mode

Files changed:
asiLogin.w
- ensure DBs are disconnected regardless of how user exits this program
prerun.p
- move checkuserlocked to userlogin.p
- add "mode" as parameter for userlogin (to capture/display in userlog record)
browsers/userlog.w
- add additional fields to browse
custom/monitor.w
- add Run Now button for user monitor only
system/userlogin.p
- capture mode chosen during login and PID of user client process
- capture int value of "enforceUserCount" for strong disconnect
- test for user locked by administrator
- fix issue with multi-session checker not respecting Exit Application button
- if user chooses multi-sessions, don't allow if other session(s) have records locked - new functionality
- if user chooses to log out other sessions AND NK1 allows it, force kill other sessions
system/wSession.w
- send variable notice that user chose to Exit login
userControl/monitor.w
- allow Run Now button to process records
- function fGetDtTmFromConnectTime  - converts DB log time to usable variable
- remove test for valid NU3 record - if not there remove userlog
- build time lag checker for users logged in who haven't yet acquired a userlog record
nosweat.i
- ensure DBs disconnected regardless of exit method
